### ISSUE TITLE
Feature request: -R and -T by Chrom Pos Ref and Alt

### ISSUE THREAD

**Issue by drhodesbrc** [samtools/bcftools#1320]
Hi,

When filtering a bcf with -R or -T, it would be useful if it was possible to do so using ref and alt information in addition to chrom-pos.
Currently I am having to save a bcf header, then append variants after piping through awk to match alt/ref. 

This would especially useful for decomposed multiallelic data where many lines will share the same chrom pos.

Thanks

**Comment by pd3** [comment]
This should not be so difficult to add for -T, the synced vcf reader in htslib is already able to do that.

**Comment by jaredo** [comment]
I just had this issue myself. I *think* this does what we want:

```
bcftools isec my.bcf sites_to_keep.vcf.gz -n=2 -w1 -Ob -o out.bcf
```

or am I missing something?

**Comment by pd3** [comment]
Yes, thank you, this is a good workaround. (Though different from the request of making this part of the -R  functionality.)
